### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by vitorbaptista**

### CODE REVIEW

File: agents/s3.js [hunk 1]
```diff
@@ -25,9 +25,10 @@ function _s3UploadSignature(s3Config, policyBase64) {
 }
 
 function _s3UploadPolicy(s3Config, key, credential, additionalConditions) {
+  const currentTime = new (Date)().getTime();
   return {
     // 30 minutes into the future
-    expiration: new Date((new Date).getTime() + (30 * 60 * 1000)).toISOString(),
+    expiration: new (Date)(currentTime + (30 * 60 * 1000)).toISOString(),
```
Comment (nightsh) [inline agents/s3.js]: Oh, nice!

File: config/index.js [hunk 2]
```diff
@@ -82,8 +85,10 @@ const config = {
...
 const knexConfig = require(path.join(__dirname, '..', './knexfile'))[env];
 const knex = require('knex')(knexConfig);
-config.bookshelf = require('bookshelf')(knex);
-config.bookshelf.plugin('registry');
-config.bookshelf.plugin('virtuals');
+const bookshelf = require('bookshelf')(knex);
+
+bookshelf.plugin('registry');
+bookshelf.plugin('virtuals');
+config.bookshelf = bookshelf;
```
Comment (nightsh) [inline config/index.js]: Should we make a convention out of this? If so, I agree with it.

File: gulpfile.js [hunk 3]
```diff
@@ -15,8 +18,8 @@ const paths = {
 
 const sassIncludePaths = [
   path.join(__dirname, 'node_modules'),
-  ...require('bourbon').includePaths,
-  ...require('bourbon-neat').includePaths,
+  ...bourbon.includePaths,
+  ...neat.includePaths,
```
Comment (nightsh) [inline gulpfile.js]: 👍

File: gulpfile.js [hunk 4]
```diff
@@ -5,6 +5,9 @@ const gulp = require('gulp');
 const plugins = require('gulp-load-plugins')();
 const del = require('del');
 const webpack = require('webpack-stream');
+const bourbon = require('bourbon');
+const neat = require('bourbon-neat');
+const webpackConfig = require('./webpack.config.js');
```
Comment (nightsh) [inline gulpfile.js]: 👍

File: package.json [hunk 5]
```diff
@@ -36,62 +36,63 @@
   },
   "homepage": "https://github.com/opentrials/opentrials#readme",
   "devDependencies": {
-    "coveralls": "^2.11.6",
-    "eslint": "^2.7.0",
-    "eslint-config-airbnb": "^7.0.0",
+    "coveralls": "^2.11.12",
+    "eslint": "^3.5.0",
+    "eslint-config-airbnb": "^11.1.0",
```
Comment (nightsh) [inline package.json]: Wow, that's a big linting bump!

**Comment by nightsh** [comment]
Awesome work on this, Vitor! Looks like I've a load of changelogs to read now :D 

I will test a clean setup (w/out involving `knex` stuff) and raise a thumb if all goes well.

I'd recommend squashing before merging though.

File: config/index.js (hunk 2 above)
Comment (vitorbaptista) [inline config/index.js]: To be honest, I don't care much. I changed because it was the new default and was easy enough.

Comment (nightsh) [inline config/index.js]: Same here, hence the agreement. But it's nice to have one way to do it that is consistent throughout the code base.

**Comment by nightsh** [comment]
OK, good to go! 👍

**Pull request merged**